exonum-java-binding/CHANGELOG.md

```diff
@@ -21,8 +21,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 The main feature of the release is enhanced support of dynamic services and
 their lifetime. Service instances might be stopped and resumed now. While
-stopped, the service artifact can be upgraded, which allows services API updates
-and synchronous data migration.
+stopped, the service artifact can be upgraded, which allows services API and
+business logic updates.
```

Comment:
I am not sure even this will work: doesn't returning `Err` as the migration script say "no migration supported at all"?

Reply:
Well, the user technically can swap artifact without changing its version. I'd remove it though.